test/TestExpectations.json

```diff
@@ -395,6 +395,12 @@
     "parameters": ["cdp", "firefox"],
     "expectations": ["SKIP"]
   },
+  {
+    "testIdPattern": "[input.spec] input tests input *",
```

Comment:
this pattern looks to be too broad?

Reply:
It actually only tests file upload lol

Reply:
let's rename `input` to `ElementHandle.uploadFile` in the test file?